### ISSUE TITLE
High WER on Librispeech

### ISSUE THREAD

**Issue by pchampio** [YiwenShaoStephen/pychain_example#8]
I've adapted the mini_librispeech example to work with the librispeech 100h datasplit.
And my WER is pretty high, 29.78%.
From this report https://arxiv.org/pdf/2010.03466.pdf I'm supposed to get ~20 of WER on dev_other.

Model:
```
    --layers 12 \
    --hidden-dims 1024 1024 1024 1024 1024 1024 1024 1024 1024 1024 512 384 \
    --kernel-sizes 3 3 3 3 3 3 3 3 3 3 3 3 \
    --dilations    1 1 1 1 1 1 1 1 1 3 3 3 \
    --strides      1 1 1 1 1 1 1 1 1 1 1 3 \

unit=phone # phone/char
type=bi # mono/bi
```
Logs:
```
Validation: [18][194/195]       Time 4.547 (1.045)      Loss 0.1216 (0.2122)
Train: [19][0/1784]     Lr: 0.00025     Time 2.651 (2.651)      Loss 0.0742 (0.0742
```

WER:
```
%WER 29.19 [ 9040 / 30965, 813 ins, 1924 del, 6303 sub ] exp/tdnn_biphone/decode/dev_other/tglarge/wer_9_0.0
```

Have you any idea why I can't get comparable result to https://arxiv.org/pdf/2010.03466.pdf
Thanks !

**Comment by YiwenShaoStephen** [comment]
I think the difference is more about the hyper-parameter tuning. You could probably try the same settings as in their paper, such as model structure and learning rate scheduling. Personally I haven't tuned with librispeech 100h so I'm afraid I can't give you too much advice on heuristics. You could also email them to see if they have some specific suggestions on that.

**Comment by pchampio** [comment]
Thanks!
